### PR TITLE
creatore refresh after code change

### PR DESCRIPTION
**Pull request by DamianAexol**
code debounced save, changes in code are visible in activeNode now, paint node opacity bugfix after code change, removed keyboard save

### CODE REVIEW

File: packages/editor/src/Graf/Graf.tsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ export const Graf: React.FC = () => {
         ? findInNodes(tree.nodes, selectedNode.field) ||
           findInNodes(libraryTree.nodes, selectedNode.field)
         : undefined,
-    [selectedNode?.field],
+    [selectedNode?.field, tree],
```
Comment (DamianAexol) [inline packages/editor/src/Graf/Graf.tsx:140]: activeNode is being refreshed now after tree change

File: packages/editor/src/editor/code/CodePane.tsx [hunk 2]
```diff
@@ -44,13 +44,8 @@ export const CodePane = (props: CodePaneProps) => {
...
 
   useEffect(() => {
-    const mounted = mount({
-      [KeyboardActions.Save]: () => {
-        onChange(temporaryString);
-      },
-    });
-    return mounted.dispose;
-  }, [temporaryString]);
+    onChange(debouncedTemporaryString);
```
Comment (DamianAexol) [inline packages/editor/src/editor/code/CodePane.tsx:47]: thanks to this, creator now sees changes made to the code with delay specified in useDebouncedValue hook

File: packages/editor/src/shared/components/PaintNodes/PaintNode.tsx [hunk 3]
```diff
@@ -187,9 +187,7 @@ export const PaintNode: React.FC<NodeProps> = ({
       isError={errorNodeNames?.includes(node.name)}
       isMatchedToSearch={isMatchedToSearch}
       isNotSelected={
-        selectedNode?.field
-          ? !compareParserFields(node)(selectedNode.field)
-          : undefined
+        selectedNode?.field ? selectedNode.field.name !== node.name : undefined
```
Comment (DamianAexol) [inline packages/editor/src/shared/components/PaintNodes/PaintNode.tsx:190]: this fixes opacity glitch.

When something was changed in code the selected node had unselected opacity.

File: packages/editor/src/shared/hooks/io.tsx [hunk 4]
```diff
@@ -21,10 +21,6 @@ export const useIO = () => {
         event.preventDefault();
         on(KeyboardActions.Redo);
       }
-      if (event.key === 's' && ctrl) {
-        event.preventDefault();
-        on(KeyboardActions.Save);
-      }
```
Comment (DamianAexol) [inline packages/editor/src/shared/hooks/io.tsx:27]: this is no longer needed due to debounced useEffect save

**Review by aexol — APPROVED** [review]

**Pull request merged**